lix/packages/sdk/src/open/openLix.ts

```diff
@@ -249,16 +249,16 @@ async function handleFileChange(args: {
 
 			// no uncommitted change exists
 			if (previousUncomittedChange === undefined) {
-				const parent = (
-					await getChangeHistory({
-						atomId: value.id,
-						depth: 1,
-						db: args.db,
-						fileId,
-						pluginKey: plugin.key,
-						diffType: diff.type,
-					})
-				)[0];
+				// const parent = (
```

Comment:
this is important, if it crashes it needs to be fixed not disabled. what exactly is the issue eror, so i can look into it?

Reply:
You can look into it. But please unblock @NiklasBuchfink and @NilsJacobsen tmr with Fink2. 

The issue is that there was no change for the query that is executed here. I saw that you intend to create a "0 commit" for a change but that does not account for all sorts of changes a plugin might safe. In other words, I don't think you can create a 0 commit during creation of lix, only during runtime if you encounter a change is has no parent id. At that point though, why bother about a 0 commit? We could define `parent_id === undefined` as "initial change"/start of the history.

Reply:
@samuelstroschein  i fixed this with nils and niklas this morning. just for completeness, i dont intend to make a 0 commit, the 0 commit id is only needed to know a repo currently has no commits and to not have to make the parent id of commit optional just for the initial commit. for changes this is not an issue as changes without a parent happen all the time and should be a normal case.

Reply:
> the 0 commit id is only needed to know a repo currently has no commits

Why do you need a commit for that and don't query via SQL?